packages/aws-cdk-lib/aws-ssm/lib/parameter.ts

```diff
@@ -579,12 +579,16 @@ export class StringParameter extends ParameterBase implements IStringParameter {
    * and the ContextProvider will be told NOT to raise an error on synthesis
    * if the SSM Parameter is not found in the account at synth time.
    */
-  public static valueFromLookup(scope: Construct, parameterName: string, defaultValue?: string): string {
+
+  public static valueFromLookup(scope: Construct, parameterName: string, defaultValue?: string, additionalCacheKey?: string): string {
```

Comment:
`defaultValue` is borderline (should probably have been in an options struct already), but I'd really like the `additionalCacheKey` to be part of an options struct.

Reply:
I'm going to assume I shouldn't touch defaultValue since it'd be an API breaking change regardless